src/main/java/org/folio/rest/client/WebClientProvider.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.rest.client;
```

Comment:
To me this looks like over-engineering. I think a simple static initializer would do